integrations/optimizely/lib/index.js

```diff
@@ -9,8 +9,9 @@ var values = require('@ndhoule/values');
 var foldl = require('@ndhoule/foldl');
 var each = require('@ndhoule/each');
 var integration = require('@segment/analytics.js-integration');
-var push = require('global-queue')('optimizely', { wrap: false });
 var tick = require('next-tick');
+var push = require('global-queue')('optimizely', { wrap: false });
+var pushEdge = require('global-queue')('optimizelyEdge', { wrap: false });
```

Comment:
Tried to do a fancier variable assignment
```
var push = window.optimizelyEdge ? require('global-queue')('optimizelyEdge', { wrap: false }) 
                   : require('global-queue')('optimizely', { wrap: false });
```
But it wasn't passing the unit test I wrote? Would like to hear ideas for testing to get ^ in.

Reply:
Hmm, I'm guessing these import statements are executed once on the test page before the mocha suite.. So depending on whether `window.optimizelyEdge` is defined, the `push` var would statically refer to `optimizely` or `optimizelyEdge`, causing tests to fail for `optimizelyEdge` or `optimizely` respectively.

Given that `window.optimizelyEdge` probably _isn't_ defined at the beginning, pushes probably all end up going to `optimizely`, causing the `optimizelyEdge` tests to fail.

Reply:
>Given that window.optimizelyEdge probably isn't defined

Yup. I'll keep this for now so we can write test cases for Edge. Makes implementing slightly more clunky, but still readable